saskatoon/harvest/filters.py

```diff
@@ -19,15 +19,17 @@ class HarvestFilter(filters.FilterSet):
     seasons = list(set(seasons))
     seasons = sorted(seasons, key=lambda tup: tup[1])
 
-    start_date = filters.ChoiceFilter(
+    season = filters.ChoiceFilter(
+        field_name="start_date", 
```

Comment:
@farhanmasud ok I didn't know you could add the `field_name`. Could we then get away without a custom method and simply go back to setting up `lookup_expr=year` ?

Reply:
Just checked and it's working without the custom method and using `lookup_expr=year` along with `field_name`. I'm updating it. Should I remove the `lookup_expr="exact",` from the other fields as well?

Reply:
Ok great, yes they're not needed you can remove them.